h2o-hadoop/scripts/docker/cdh/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM nimmis/java:14.04-oracle-8-jdk
+
+ARG VERSION
+ARG PATH_PREFIX='.'
+
+ENV DISTRIBUTION 'cdh'
+ENV VERSION $VERSION
+ENV SETUP_H2O TRUE
+ENV START_H2O TRUE
+ENV RUN_TESTS TRUE
+ENV ENTER_BASH FALSE
+
+RUN echo "# Packages for Cloudera's Distribution for Hadoop, Version 5, on Ubuntu 14.04 amd64\n\
+deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n\
+deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n" > /etc/apt/sources.list.d/cloudera.list
+
+# Prepare Cloudera repository
+COPY ${PATH_PREFIX}/conf/cloudera.pref /etc/apt/preferences.d/cloudera.pref
+RUN wget http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key -O archive.key && sudo apt-key add archive.key && \
+    sudo apt-get update
+
+# Download necessary packages
+RUN apt-get install -y hadoop-conf-pseudo python-dev python-pip python-scipy
```

Comment:
Every dockerfile statement creates a container layer, that is why it is advised to combine related execution of logically related commands into one docker RUN statement, eg.:

  RUN apt-get install -y hadoop-conf-pseudo python-dev python-pip python-scipy & \
    pip install requests --upgrade & \
    pip install colorama future tabulate pandas